recipes/theora/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, cmake_layout
+from conan.tools.build import cross_building
+
+
+class TheoraTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    # VirtualBuildEnv and VirtualRunEnv can be avoided if "tools.env.virtualenv:auto_use" is defined
+    # (it will be defined in Conan 2.0)
+    generators = "CMakeDeps", "CMakeToolchain", "VirtualBuildEnv", "VirtualRunEnv"
```

Comment:
This is not a v1 test package 😖

Reply:
Sorry about that oversight, I asked somewhere else how these test_v1_packages are supposed to be tested locally...
I have to work late to do this stuff because you guys are all in a very different timezone.  I get stuck and have no guidance until the middle of the night.

Reply:
I see the -tf flag now

Reply:
Docs! I do my best https://github.com/conan-io/conan-center-index/blob/master/docs/developing_recipes_locally.md#testing-the-different-test__package

Every command should be in that one file